version.go

```diff
@@ -0,0 +1,4 @@
+package main
```

Comment:
I can manage this file statically if thats the preferred method which means I will just remove the version.sh script. All this does is makes it easy for me to update this particular file by using `go generate`

Regardless of the situation I will still need this file even if its generated or not